.github/workflows/concrete_python_tests_linux.yml

```diff
@@ -72,147 +75,140 @@ jobs:
           options: >-
             -v ${{ github.workspace }}:/concrete
             -v ${{ github.workspace }}/build:/build
-            -v ${{ env.SSH_AUTH_SOCK }}:/ssh.socket
-            -e SSH_AUTH_SOCK=/ssh.socket
           shell: bash
           run: |
-            rustup toolchain install nightly-2024-09-30
```

Comment:
Would be better to use `env:` variable for this, something like `RUST_TOOLCHAIN: nightly-2024-09-30`

Reply:
removed was useless